externs/svg-externs.js

```diff
@@ -0,0 +1,10367 @@
+/**
+ * @fileoverview Generated by closureidl 0.2dev closureidl-parser-webkit 0.2dev dart-idl 7592.
```

Comment:
I've found that this extern is already in closure at https://github.com/google/closure-compiler/commits/master/contrib/externs/svg.js so we could use that.
If you want, I will add a dependency to closure-compiler's npm package to package.json, and the extern file can be read from there. What do you think @rachel-fenichel ? I don't know if it is possible to include it without its source via the closure's rest api (referencing its name only) since this is not documented.
Or, alternatively, it can be moved to the demo's directory, to show how externs would be managed if somebody wants to use some other library than Closure Library.